isamples_metadata/Transformer.py

```diff
@@ -134,21 +144,42 @@ def sample_sampling_purpose(self) -> str:
         """The samplingPurpose of the sample registrant in source_record"""
         pass
 
+    @staticmethod
+    def _rule_based_confidence_list_for_categories_list(category_list: list) -> typing.Optional[list]:
+        if category_list is None:
+            return None
+        confidences = []
+        for _ in category_list:
+            confidences.append(Transformer.RULE_BASED_CONFIDENCE)
+        return confidences
+
     @abstractmethod
     def has_context_categories(self) -> typing.List[str]:
         """Map from the source record into an iSamples context category"""
         pass
 
```

Comment:
New default implementations for all confidences -- will set things to 1.0 unless overridden.